config/environments/test.rb

```diff
@@ -54,15 +54,16 @@
   # Raise exceptions on deprecation notices.
   config.active_support.deprecation = :raise
 
-  if ENV.fetch('TEST_LOGGING', nil) == '1'
-    # :nocov:
-    config.log_level = :debug
-    config.active_record.verbose_query_logs = true
+  config.active_record.verbose_query_logs = true
```

Comment:
I'm setting `verbose_query_logs = true` always, because we don't write the logs at all in the default condition (when `TEST_LOGGING` is not `1`), so presumably it doesn't negatively impact performance then? And when I do have logging on, then I think I will always want the verbose query logs.